src/runtime_src/xdp/profile/plugin/aie_debug/aie_debug_plugin.cpp

```diff
@@ -350,33 +358,49 @@ namespace xdp {
     // Open the writer for this device
     struct xclDeviceInfo2 info;
     xclGetDeviceInfo2(handle, &info);
-    std::string deviceName { info.mName };
-    // Create and register writer and file
-    std::string outputFile = "aie_status_" + deviceName + ".json";
+    std::string devicename { info.mName };
```

Comment:
Again camel case here and below (fileName, aieWriter, aieshimWriter, etc.)

Reply:
all local variables in this functions aren't camel case.